tactic/fin_cases.lean

```diff
@@ -7,7 +7,7 @@ Case bashing on `fin n`, for explicit numerals `n`.
 -/
 import tactic.linarith
 
-def fin_val {m n : ℕ} (h : m < n) : (⟨m, h⟩ : fin n).val = m := rfl 
+def fin_val {m n : ℕ} (h : m < n) : (⟨m, h⟩ : fin n).val = m := rfl
```

Comment:
use `fin.eta` (if not, move it to `data/fin.lean` and it should be a lemma)

Reply:
I made
```
@[simp] def mk_val {m n : ℕ} (h : m < n) : (⟨m, h⟩ : fin n).val = m := rfl
```
as nothing equivalent seemed to exist.